cluster/acsk.go

```diff
@@ -69,7 +69,7 @@ func (c *ACSKCluster) ListNodeNames() (nodes pkgCommon.NodeNames, err error) {
 		return
 	}
 
-	r, err := getClusterDetails(client, c.modelCluster.ACSK.ClusterID)
+	r, err := getClusterDetails(client, c.modelCluster.ACSK.ClusterIdentifier)
```

Comment:
I would call this ProviderClusterID to reflect the source of this id better.

Reply:
👍 done